cpp/arcticdb/arrow/test/test_arrow_write.cpp

```diff
@@ -173,11 +173,12 @@ TYPED_TEST(ArrowDataToSegmentNumeric, MultipleRecordBatches) {
         ASSERT_EQ(buffer.block_offsets().size(), rows_per_batch.size() + 1);
         size_t bytes{0};
         for (size_t idx = 0; idx < rows_per_batch.size(); ++idx) {
-            ASSERT_TRUE(buffer.blocks()[idx]->is_external());
-            ASSERT_EQ(buffer.blocks()[idx]->bytes(), rows_per_batch[idx] * sizeof(TypeParam));
-            ASSERT_EQ(buffer.blocks()[idx]->offset_, bytes);
+            ASSERT_EQ(buffer.blocks()[idx]->get_type(), MemBlockType::EXTERNAL_WITH_EXTRA_BYTES);
+            ASSERT_EQ(buffer.blocks()[idx]->physical_bytes(), rows_per_batch[idx] * sizeof(TypeParam));
+            ASSERT_EQ(buffer.blocks()[idx]->logical_size(), rows_per_batch[idx] * sizeof(TypeParam));
```

Comment:
nit: These two names look odd side by side. Does it make sense to either rename physical_bytes to `physical_size` or `logical_size` to `logical_bytes`